docs_user/modules/proc_updating-the-object-storage-endpoints.adoc

```diff
@@ -1,8 +1,9 @@
 [id="updating-the-object-storage-endpoints_{context}"]
 
-= Updating the object-store endpoints
+= Updating the object storage endpoints
 
-The object-storage endpoints still point to the original virtual IP address (VIP) that is owned by pacemaker. You must update the object-store endpoints because other services stll use the original VIP, and you reserved a new VIP on the same network.
+The object-storage endpoints still point to the original virtual IP address (VIP) that is owned by Pacemaker. 
+You must update the object-storage endpoints to point to the new VIP because other services stll use the original VIP, and you reserved a new virtual IP address (VIP) that you reserved on the same network.
 
```

Comment:
1. Does “object-storage” refer to the “Object storage service”? If so, we should refer to the end points as “Object storage service endpoints”.
2. What does “other services” refer to?
3. I’m not sure what “same network” refers to either. The same network that the other services use?
4. Proposed edit for the entire intro: You must update the object-storage endpoints to point to the new virtual IP address (VIP) that you reserved on the same network.

Reply:
1. Yes
2. other services: clients (third party, e.g. s3) that consume `Object storage service` 
3. We're re-deploying RGW ingress (it's an HAproxy behind the scenes) using the same network (e.g. external network) but different VIP (Virtual Ip Address). So the scenario I'm referring to is where the service is moved to a different IP within the same network. Does this make sense?
4. Works for me!

Reply:
@fmount This is the new intro to "Updating the Object storage service endpoints":

"You must update the {object_storage_first_ref} endpoints to point to the new virtual IP address (VIP) that you reserved on the same network that you used to deploy RGW ingress."

Reply:
Works for me ^, Thank you!